src/main/java/org/apache/datasketches/quantiles/ItemsSketch.java

```diff
@@ -237,311 +258,369 @@ static <T> ItemsSketch<T> copy(final ItemsSketch<T> sketch) {
 
   /**
-   * Gets the lower bound of the value interval in which the true quantile of the given rank
-   * exists with a confidence of at least 99%.
-   * @param fraction the given normalized rank as a fraction
-   * @return the lower bound of the value interval in which the true quantile of the given rank
-   * exists with a confidence of at least 99%. Returns NaN if the sketch is empty.
+   * Returns the minimum quantile of the stream. This is provided for convenience, but is distinct from the smallest
+   * quantile retained by the sketch algorithm.
+   *
+   * <p>If the sketch is empty this returns null.</p>
+   *
+   * @return the minimum quantile of the stream
    */
-  public T getQuantileLowerBound(final double fraction) {
-    return getQuantile(max(0, fraction - Util.getNormalizedRankError(k_, false)));
+  public T getMinItem() { return minItem_; }
+
+  /**
+   * Same as {@link #getPMF(Object[], QuantileSearchCriteria) getPMF(splitPoints, INCLUSIVE)}
+   * @param splitPoints an array of <i>m</i> unique, monotonically increasing quantiles
+   * @return a PDF array of m+1 probability masses as doubles on the interval [0.0, 1.0).
+   */
+  public double[] getPMF(final T[] splitPoints) {
+    return getPMF(splitPoints, INCLUSIVE);
   }
 
   /**
-   * This is a more efficient multiple-query version of getQuantile().
+   * Returns an approximation to the Probability Mass Function (PMF) of the input stream
+   * as an array of double probability masses on the interval [0.0, 1.0],
+   * given a set of splitPoints.
+   * The sum of the probability masses in the returned array is always 1.0.
+   *
+   * <p>The resulting approximations have a probabilistic guarantee that can be obtained from the
+   * getNormalizedRankError(true) function.
+   *
+   * <p>If the sketch is empty this returns null.</p>
+   *
+   * @param splitPoints an array of <i>m</i> unique, monotonically increasing quantiles
+   * that divide the real number line into <i>m+1</i> consecutive non-overlapping intervals.
+   * It is not necessary to include either the minimum or maximum quantiles
+   * of the input stream in these split points.
+   *
+   * <p>If searchCrit is INCLUSIVE, the definition of an "interval" is
+   * exclusive of the left splitPoint and
+   * inclusive of the right splitPoint.</p>
    *
-   * <p>This returns an array that could have been generated by using getQuantile() with many
-   * different fractional ranks, but would be very inefficient.
-   * This method incurs the internal set-up overhead once and obtains multiple quantile values in
-   * a single query. It is strongly recommend that this method be used instead of multiple calls
-   * to getQuantile().
+   * <p>If searchCrit is EXCLUSIVE, the definition of an "interval" is
+   * inclusive of the left splitPoint and
+   * exclusive of the right splitPoint.</p>
    *
-   * <p>If the sketch is empty this returns null.
+   *<p>The left "splitPoint" for the lowest interval is the
```

Comment:
I don't understand this paragraph. The user provides split points, and we say just above that it is not necessary to include min and max.

Reply:
My mistake.  This Javadoc for getPMF(splitPoints, searchCrit) was supposed to be a near exact copy of the same Javadoc for this same methods for floats and doubles, but clearly it didn't get updated.  I will fix.

Reply:
I would not understand this paragraph with respect to either floats or doubles